addToCollection.php

```diff
@@ -0,0 +1,104 @@
+<?php
+require_once 'functions.php';
+
+$db = getDB();
+$cleansedArray = cleanseData($_POST);
+
+if(!empty($cleansedArray)) {
+    if (checkItemExists($cleansedArray, $db) != []) {
```

Comment:
edit checkItemExists so that it just returns a true/false - makes it easier for use here and any reuse in the future as will likely make up a conditional whenever its used